src/test/java/com/fasterxml/jackson/dataformat/xml/ser/Jdk8StreamSerialization302Test.java

```diff
@@ -0,0 +1,128 @@
+package com.fasterxml.jackson.dataformat.xml.ser;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlTestBase;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Stream;
+
+// [dataformat-xml#302] : Unable to serialize top-level Java8 Stream
+public class Jdk8StreamSerialization302Test extends XmlTestBase {
+
+    final ObjectMapper OBJECT_MAPPER = new XmlMapper();
+
+    public static class StreamWrapper329 {
+        private Stream<String> data;
+
+        @JacksonXmlElementWrapper(localName = "elements")
+        @JacksonXmlProperty(localName = "element")
+        public Stream<String> getData() {
+            return data;
+        }
+
+        public void setData(Stream<String> data) {
+            this.data = data;
+        }
+    }
+
+    public static class CollectionWrapper329 {
+        private Collection<String> data;
+
+        @JacksonXmlElementWrapper(localName = "elements")
+        @JacksonXmlProperty(localName = "element")
+        public Collection<String> getData() {
+            return data;
+        }
+
+        public void setData(Collection<String> data) {
+            this.data = data;
+        }
+    }
+
+    public static class IteratorWrapper329 {
+        private Iterator<String> data;
+
+        @JacksonXmlElementWrapper(localName = "elements")
+        @JacksonXmlProperty(localName = "element")
+        public Iterator<String> getData() {
+            return data;
+        }
+
+        public void setData(Iterator<String> data) {
+            this.data = data;
+        }
+    }
+
+    public void testCollectionSerialization() throws JsonProcessingException {
```

Comment:
Let's change these to `throws Exception` since `JsonProcessingException` is removed from 3.0 and `JacksonException` (that exists) does not extend `IOException` any longer.
So use of plain `Exception` allows cleaner merging.